library/swift/EngineBuilder.swift

```diff
@@ -169,6 +171,28 @@ public final class EngineBuilder: NSObject {
     return self
   }
 
+  /// Set a closure to be called when the engine's logger logs.
+  ///
+  /// - parameter closure: The closure to be called.
+  ///
+  /// - returns: This builder.
+  @discardableResult
+  public func setonEngineLog(closure: @escaping (String) -> Void) -> EngineBuilder {
+    self.onEngineLog = closure
+    return self
+  }
+
+  /// Set a closure to be called when the engine's logger flushes.
+  ///
+  /// - parameter closure: The closure to be called.
+  ///
+  /// - returns: This builder.
+  @discardableResult
+  public func setonEngineFlush(closure: @escaping () -> Void) -> EngineBuilder {
```

Comment:
```suggestion
  public func setOnEngineFlush(closure: @escaping () -> Void) -> EngineBuilder {
```
If we decide to keep this